custom-lambda-authorizer/index.mjs

```diff
@@ -0,0 +1,103 @@
+import jwt from 'jsonwebtoken';
```

Comment:
Please add the license.

Reply:
fixed with https://github.com/wso2/samples-apim/pull/149/commits/e4c87b5b1ee91e652c7076af5e4f51e09edd7638